1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_1_Fizzbuzz/src/index.spec.ts

```diff
@@ -1,5 +1,41 @@
-
+import fizzbuzz from './fizzbuzz';
 describe("fizzbuzz", () => {
 
+  it("return error if number is less than 1", () => {
+    expect(() => fizzbuzz(0)).toThrow();
+  });
+
+  it("return error if number is more than 100", () => {
+    expect(() => fizzbuzz(101)).toThrow();
+  });
+
+  it("returns a string", () => {
+    expect(typeof fizzbuzz(5)).toBe('string');
```

Comment:
Since `fizzBuzz` returns a `string`, which is a static test/guarantee of the return type, perhaps this test can be omitted? I can see also see a benefit in this test being a safe-guard for, say, unintentional changes in the return type e.g. changing the return type to `any`, but I guess that reason for change would've had to come from somewhere and would impact further requirements and tests.

It's a fine line :)

Reply:
I think its good to go with the safer version of it